test/mocks/auth/MockAuthActions.scala

```diff
@@ -53,10 +61,15 @@ trait MockAuthActions extends
   override def beforeEach(): Unit = {
     super.beforeEach()
     disableAllSwitches()
-    Play.stop(fakeApplication())
+//    Play.stop(fakeApplication())
```

Comment:
if this code is not required, we can drop it?

Reply:
This was the fix for the tests length. I will remove it in my next PR.